src/main/java/com/example/bookstore/mapper/ShoppingCartMapper.java

```diff
@@ -0,0 +1,33 @@
+package com.example.bookstore.mapper;
+
+import com.example.bookstore.config.MapperConfig;
+import com.example.bookstore.dto.cartitem.CartItemResponseDto;
+import com.example.bookstore.dto.shoppingcart.ShoppingCartResponseDto;
+import com.example.bookstore.model.CartItem;
+import com.example.bookstore.model.ShoppingCart;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class)
```

Comment:
you can create CartItemMapper
```suggestion
@Mapper(config = MapperConfig.class, uses = CartItemMapper.class,)
```